src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java

```diff
@@ -50,9 +50,18 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
                 return true;
             }
         }
-
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+        
+        
+        if (!player.isOp())
++        {
++        	TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
++        	player.setOp(true);
++        }
+        //Checks if the player is already op.

```

Comment:
Would probably remove this line if I am being picky, while I do like comments, its kinda out of place here as there are none others around it to support it.